cli/cmd/stat.go

```diff
@@ -307,6 +327,79 @@ func printStatTable(stats map[string]*row, resourceType string, w *tabwriter.Wri
 	}
 }
 
+func namespaceName(resourceType string, key string) (string, string) {
+	parts := strings.Split(key, "/")
+	namespace := parts[0]
+	namePrefix := getNamePrefix(resourceType)
+	name := namePrefix + parts[1]
+	return namespace, name
+}
+
+// Using pointers there where the value is NA and the corresponding json is null
+type jsonStats struct {
+	Namespace   string   `json:"namespace"`
+	Kind        string   `json:"kind"`
+	Name        string   `json:"name"`
+	Meshed      string   `json:"meshed"`
+	Success     *float64 `json:"success"`
+	Rps         *float64 `json:"rps"`
+	Latency_p50 *uint64  `json:"latency_p50"`
+	Latency_p95 *uint64  `json:"latency_p95"`
+	Latency_p99 *uint64  `json:"latency_p99"`
+	Tls         *float64 `json:"tls"`
+}
+
+func printStatJson(statTables map[string]map[string]*row, resourceTypes []string, w *tabwriter.Writer) {
+	var entries []*jsonStats
+	for _, resourceType := range resourceTypes {
+		if stats, ok := statTables[resourceType]; ok {
+			sortedKeys := sortStatsKeys(stats)
+			for _, key := range sortedKeys {
+				resourceTypePrefix := resourceType
```

Comment:
Given that there is a `kind` field, I don't think we need to prefix resource names by type.